CONTRIBUTING.rst

```diff
@@ -160,3 +160,16 @@ language, you must:
     copy-and-paste to create the corresponding new test.
 
     If in doubt, ask Dateparser maintainers for help.
+
+Updating the List of Supported Languages and Locales
+----------------------------------------------------
+
+Whenever the content of
+``dateparser.data.languages_info.language_locale_dict`` is modified, use
+``dateparser_scripts/update_supported_languages_and_locales.py`` to update 
+the corresponding documentation table::
+
+    python3 -m venv venv
+    . venv/bin/activate
+    pip install -e .
```

Comment:
do we need to add these three lines? I think it's too verbose and it shouldn't be necessary. @Gallaecio

Reply:
@Gallaecio 
In order to merge it, let me know if you change this or the reasons if you leave it :)